### ISSUE TITLE
Lights still get brighter

### ISSUE THREAD

**Issue by broyuken** [basnijholt/adaptive-lighting#13]
I was in my living room this morning with my lights in night mode (1% brightness). The house went into day mode which turns off AL night mode so the lights got brighter which is fine, that’s what’s it’s supposed to do. However I then went to the wall and turned them back down to 1%, but a few seconds later they were taken back over by AL and made bright again. 

I had to turn the brightness up and then down again within HA to get them to stop. Note, I have both take over control and the manual changes enabled for these lights.

It seems like light.turn_off service calls may not make AL understand the light has been manually controlled, only light.turn on?

**Comment by basnijholt** [comment]
Could it be that the same lights are in two different AL entities?

**Comment by broyuken** [comment]
Nope, this is only in my living room AL

**Comment by basnijholt** [comment]
It would be really helpful if you could have a log file such that I can better understand what is going on :)

**Comment by broyuken** [comment]
I'll see if I can turn on debug logging again and try to reproduce it. The opposite happened this morning. Sleep mode was on so the lights were at 1%, I turned the lights up in the kitchen this time from lovelace, then a minute later they dimmed back to 1%. I had to disable sleep mode to get them back to a reasonable brightness. The living room lights were zigbee lights and the kitchen was a zwave switch, so no correlation there.
